StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1309UnitTests.cs

```diff
@@ -132,6 +132,23 @@ public class Foo
             await this.VerifyCSharpFixAsync(testCode, fixedCode).ConfigureAwait(false);
         }
 
+        // This a regression test for issue #2360.
+        [Fact]
+        public async Task VerifyThatAFieldStartingWithAnUnderscoreAndADigitIsNotAffectedByCodeFixAsync()
```

Comment:
❓ Can you add a test verifying that a field `_int` is correctly refactored to `@int`? I want to make sure that `IsValidIdentifier` isn't preventing us from using an escaped identifier.